test/accessors/SimulateTxAccessor.spec.ts

```diff
@@ -6,7 +6,7 @@ import { buildContractCall } from "../../src/utils/execution";
 describe("SimulateTxAccessor", () => {
     const setupTests = deployments.createFixture(async ({ deployments }) => {
         await deployments.fixture();
-        const signers = await ethers.getSigners();
+        const signers = await hre.ethers.getSigners();
```

Comment:
the change here (and a similar change in other files) is unrelated to zksync: it was in the original PR and I decided to keep it because this way the invocation is more specific and less likely to cause a conflict with a global import like this `import { ethers } from 'ethers'`